gengokit/httptransport/templates_test.go

```diff
@@ -62,6 +62,8 @@ func TestGenClientEncode(t *testing.T) {
 // the http request (path, query, and body).
 func EncodeHTTPSumZeroRequest(_ context.Context, r *http.Request, request interface{}) error {
 	fmt.Printf("Encoding request %v\n", request)
+	strval := ""
+	_ = strval
 	req := request.(*pb.SumRequest)
 	_ = req
```

Comment:
Depending on what http annotations are used, `req` will be used or unused. Same reason as https://github.com/TuneLab/go-truss/pull/136#discussion_r97655106